format/spec.md

```diff
@@ -654,17 +654,17 @@ The `first_row_id` is only inherited for added data files. The inherited value m
-| _required_ | _required_ | _required_ | **`timestamp-ms`**           | A timestamp when the snapshot was created, used for garbage collection and table inspection                                        |
-| _optional_ | _required_ | _required_ | **`manifest-list`**          | The location of a manifest list for this snapshot that tracks manifest files with additional metadata                              |
-| _optional_ |            |            | **`manifests`**              | A list of manifest file locations. Must be omitted if `manifest-list` is present                                                   |
-| _optional_ | _required_ | _required_ | **`summary`**                | A string map that summarizes the snapshot changes, including `operation` as a _required_ field (see below)                         |
-| _optional_ | _optional_ | _optional_ | **`schema-id`**              | ID of the table's current schema when the snapshot was created                                                                     |
-|            |            | _optional_ | **`first-row-id`**           | The first `_row_id` assigned to the first row in the first data file in the first manifest, see [Row Lineage](#row-lineage) |
+| v1         | v2         | v3         | Field                        | Description                                                                                                                         |
+| ---------- | ---------- |------------|------------------------------|-------------------------------------------------------------------------------------------------------------------------------------|
+| _required_ | _required_ | _required_ | **`snapshot-id`**            | A unique long ID                                                                                                                    |
+| _optional_ | _optional_ | _optional_ | **`parent-snapshot-id`**     | The snapshot ID of the snapshot's parent. Omitted for any snapshot with no parent                                                   |
+|            | _required_ | _required_ | **`sequence-number`**        | A monotonically increasing long that tracks the order of changes to a table                                                         |
+| _required_ | _required_ | _required_ | **`timestamp-ms`**           | A timestamp when the snapshot was created, used for garbage collection and table inspection                                         |
+| _optional_ | _required_ | _required_ | **`manifest-list`**          | The location of a manifest list for this snapshot that tracks manifest files with additional metadata                               |
+| _optional_ |            |            | ~~**`manifests`**~~          | ~~A list of manifest file locations. Must be omitted if `manifest-list` is present.~~ (**Deprecated**: use `manifest-list` instead) |
```

Comment:
what does the `~~` do here? Also looks like the `|` at the end of the table aren't aligned anymore in the last 3 lines

Reply:
It does a strikethrough, similar to the ones that are deprecated from the `data_file`:
![image](https://github.com/user-attachments/assets/b834e7f0-20c1-40dd-8f2e-06180f7c663d)

I've fixed the indentation, thanks for spotting that 👍

Reply:
ah nice, thanks for posting the picture. that makes it easier